src/test/java/picard/vcf/VcfToIntervalListTest.java

```diff
@@ -0,0 +1,61 @@
+package picard.vcf;
+
+import htsjdk.samtools.util.Interval;
+import htsjdk.samtools.util.IntervalList;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import picard.cmdline.CommandLineProgramTest;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by lichtens on 7/11/17.
+ */
+public class VcfToIntervalListTest extends CommandLineProgramTest {
+    private static final File TEST_RESOURCE_DIR = new File("testdata/picard/vcf");
+    public static final String smallM2VcfMore = TEST_RESOURCE_DIR.getAbsolutePath() + "/small_m2_more_variants.vcf";
+
+    public String getCommandLineProgramName() {
+        return VcfToIntervalList.class.getSimpleName();
+    }
+
+    @Test
+    public void testExcludingFiltered() throws IOException {
+        final File outputFile = File.createTempFile("vcftointervallist_", ".interval_list");
```

Comment:
these two tests are almost the same...can we data-provider them?

Reply:
Yeah - makes sense. Will do.